tests/test_model_ipython_display.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import requests
+from pringles.models import Coupled
+from .utils import make_queue_top_model_with_events
+
+
+@pytest.fixture(autouse=True)
+def _start_web_display_backend():
+    _start_server()
```

Comment:
In here, mock the `_get_static_files_path` to move test static files to another folder.